widget/ui/src/icons/index.ts

```diff
@@ -7,6 +7,7 @@ export { default as ChevronDownIcon } from './ChevronDown';
 export { default as ChevronLeftIcon } from './ChevronLeft';
 export { default as ChevronRightIcon } from './ChevronRight';
 export { default as ChevronUpIcon } from './ChevronUp';
+export { default as ClearIcon } from './Clear';
```

Comment:
I think this is not required. you could use close icon.
also make sure that both clear and copy icon have same hover effect/color in confirm wallet page.